scripts/erc20TokenBridgeDeployment.ts

```diff
@@ -219,6 +219,11 @@ export const createTokenBridge = async (
   const l2ProxyAdmin =
     await l1TokenBridgeCreator.getCanonicalL2ProxyAdminAddress(childChainId)
 
+  const l1MultiCall = await l1TokenBridgeCreator.l1MultiCall()
```

Comment:
Typo in function name
```suggestion
  const l1MultiCall = await l1TokenBridgeCreator.l1Multicall()
```